test/overlay.html

```diff
@@ -347,6 +360,35 @@
         });
       });
     });
+
+    describe('frontmost', () => {
+      var dialog;
+      beforeEach(() => {
+        dialog = fixture('front-most');
+      });
+
+      it('should bring to frontmost on mousedown', (done) => {
+        dialog.opened = true;
+        const dialogOverlay = dialog.$.overlay;
+        const datePicker = dialog.$.overlay.querySelector('vaadin-date-picker');
+    
+        const datePickerPosition = datePicker.getBoundingClientRect();
+        const datePickerOverlay = datePicker.$.overlay;
+
+        datePicker.addEventListener('opened-changed', () => {
+          Polymer.RenderStatus.afterNextRender(datePicker, () => {
+            datePicker.dispatchEvent(new CustomEvent('mousedown', {bubbles: true, composed: true}));
```

Comment:
Also dispatch a "touchstart" event, since the code has changed logic for it

Reply:
done